src/components/MdDatepicker/MdDatepicker.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <md-field :class="['md-datepicker', { 'md-native': !this.mdOverrideNative }]" md-clearable>
+  <md-field :class="['md-datepicker', { 'md-native': !this.mdOverrideNative }]" :md-clearable="mdClearable" @md-clear="onClear">
```

Comment:
What about

```
<md-field
  :class="['md-datepicker', { 'md-native': !this.mdOverrideNative }]"
  v-bind="$props"
  v-on="listeners">
```

Then add a computed

```
    computed: {
      listeners () {
        return {
          ...this.$listeners,
          // here you can add override listeners
        }
      },
    },
```

Reply:
+1 for this solution!

Reply:
I think the current way is fine, to stay consistent with the existing api